lib/transforms/import-declarations.js

```diff
@@ -28,6 +28,8 @@ function transformer(file, api) {
           var inTestsPrefix = appName + '/app/tests/';
           if (importPath.startsWith(inTestsPrefix)) {
             importPath = appName + '/tests/' + importPath.substring(inTestsPrefix.length);
+          } else {
+            importPath = importPath.replace('/app/app/', '/app/');
```

Comment:
@andyhot Not sure about the logic here, there's probably a better way to do this, but in the case that `tests/helpers/resolver.js` imports `../../app/resolver.js`, `app` gets entered twice into the path, so the path here is something like `my-app/app/app/resolver`.